helpers/go/prep-go.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -eux -o pipefail
+shopt -qs failglob
+
+for opt in "$@"; do
+   optarg="$(expr "${opt}" : '[^=]*=\(.*\)')"
+   case "${opt}" in
+      --go-version=*) GOVER="${optarg}" ;;
+   esac
+done
+
+GOVER="${GOVER:?}"
+
+cd "${HOME}/sdk-go-${GOVER}"
+sdk-fetch "${HOME}/hashes" && \
+  tar --strip-components=1 -xf go${GOVER}.src.tar.gz && \
+  rm go${GOVER}.src.tar.gz
```

Comment:
You have `set -e` here, so:
```suggestion
sdk-fetch "${HOME}/hashes"
tar --strip-components=1 -xf go${GOVER}.src.tar.gz
rm go${GOVER}.src.tar.gz
```